cmd/helm/root.go

```diff
@@ -67,6 +67,7 @@ Environment variables:
 | $HELM_KUBEASUSER                   | set the Username to impersonate for the operation.                                |
 | $HELM_KUBECONTEXT                  | set the name of the kubeconfig context.                                           |
 | $HELM_KUBETOKEN                    | set the Bearer KubeToken used for authentication.                                 |
+| $HELM_BURST_LIMIT                  | set the default burst limit in the case the server contains many CRDs             |
```

Comment:
Let's both call out the default value as well how it can be disabled (-1)

Reply:
Fixed in https://github.com/helm/helm/pull/10842/commits/14daf80ca0301d987636e66a4b6d70452389b92c